langspec/xproc11/xproc.xml

```diff
@@ -1027,7 +1030,7 @@ refer to that variable will raise an error.</para>
               </listitem>
             </varlistentry>
             <varlistentry>
-              <term>Base URI</term>
+              <term>Static base URI</term>
               <listitem>
                 <para>The base URI of the element on which the expression occurs.</para>
```

Comment:
I used the same term as in XPath 3.1 specs. They say "Static Base URI", but of course we are free to choose an other term.